frontend/language/src/nb.json

```
@@ -1724,17 +1724,20 @@
   "ux_editor.no_components_selected": "Velg en side for å se forhåndsvisningen",
   "ux_editor.no_text": "Ingen tekst",
   "ux_editor.open_preview": "Åpne forhåndsvisning",
-  "ux_editor.options.code_list_only": "Denne komponenten støtter bare oppsett med forhåndsdefinerte kodelister.",
   "ux_editor.options.code_list_referenceId.description": "Her kan du legge til en referanse-ID til en dynamisk kodeliste som er satt opp i koden.",
   "ux_editor.options.code_list_referenceId.description_details": "Du bruker dynamiske kodelister for å tilpasse alternativer for brukerne. Det kan for eksempel være tilpasninger ut fra geografisk plassering, eller valg brukeren gjør tidligere i skjemaet.",
+  "ux_editor.options.modal_header_library_code_list": "Kodeliste fra biblioteket",
+  "ux_editor.options.modal_header_manual_code_list": "Kodeliste",
   "ux_editor.options.multiple": "{{value}} alternativer",
   "ux_editor.options.option_edit_text": "Rediger kodeliste",
-  "ux_editor.options.option_remove_text": "Fjern valgt kodeliste",
+  "ux_editor.options.option_remove_text": "Slett valgt kodeliste",
   "ux_editor.options.section_heading": "Valg for kodelister",
   "ux_editor.options.single": "{{value}} alternativ",
   "ux_editor.options.tab_code_list": "Velg kodeliste",
   "ux_editor.options.tab_manual": "Sett opp egne alternativer",
+  "ux_editor.options.tab_option_list_alert_title": "Komponenten refererer allerede til en egendefinert ID. Legger du til en ny kodeliste her vil du overstyre den egendefinerte ID-en.",
   "ux_editor.options.tab_referenceId": "Angi referanse-ID",
+  "ux_editor.options.tab_referenceid_alert_title": "Du har allerede referert til en kodeliste. Skriver du inn en ID, vil referansen bli slettet.",
```

```suggestion
  "ux_editor.options.tab_reference_id_alert_title": "Du har allerede referert til en kodeliste. Skriver du inn en ID, vil referansen bli slettet.",
```
If you fix this, remember to change the text key where it is used as well.

---

I'll remove this from the diff as it was part of the referenceTab PR, forgot to clean up nb.json after removing the code.